### PR TITLE
Add mp3 to the registry

### PR DESCRIPTION
**Pull request by padenot**
This is using the latest revision of the text (from [1998](https://www.iso.org/standard/26797.html)), so it's in 2.4.2.2, but generally the same as in MSE ("an mp3 frame").

Again this has the Makefile changes so that it builds, please ignore.

### CODE REVIEW

**Review by dalecurtis — APPROVED** [review]
Thanks Paul!

File: mp3_codec_registration.src.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+{{EncodedAudioChunk}} [=EncodedAudioChunk/[[internal data]]=] is expected to be
+a "frame", as described in the section 2.4.2.2 of the [[MP3]] specification.
+
+AudioDecoderConfig description {#audiodecoderconfig-description}
+================================================================
+
+{{AudioDecoderConfig.description}} is not used for this codec.
+
+The {{AudioDecoderConfig.sampleRate}} and {{AudioDecoderConfig.channelCount}}
+members are not used.
```
Comment (sandersdan) [inline mp3_codec_registration.src.html:78]: What does not used mean exactly? I do expect that UAs can still reject configurations based on their values.

File: mp3_codec_registration.src.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+The {{AudioDecoderConfig.sampleRate}} and {{AudioDecoderConfig.channelCount}}
+members are not used. 
+
+
+EncodedAudioChunk type {#encodedaudiochunk-type}
+================================================
+
+The [=EncodedAudioChunk/[[type]]=] for an {{EncodedAudioChunk}} containing
+mp3 is always "[=EncodedAudioChunkType/key=]".
+NOTE: Once the initialization has succeeded, any mp3 packet can be decoded at
```
Comment (sandersdan) [inline mp3_codec_registration.src.html:86]: Does this render correctly without the double space and second line indent?

**Review by sandersdan — APPROVED** [review]

File: mp3_codec_registration.src.html (hunk 1 above)
Comment (padenot) [reply mp3_codec_registration.src.html:78]: This was discussed last week with @dalecurtis, but we can discuss it further: https://github.com/w3c/webcodecs/issues/317#issuecomment-892801222, which is not amazing, but was considered better than making the field optional in IDL but sometimes required from the registry, that is non-normative. But maybe there's something more elegant, that we could do.

In the case of an mp3 sequence, I don't expect authors to look at the byte stream to determine the rate and channel count ahead of decoding. I expect an author to just give the packets to the decoder, having passed any value to this sample-rate and channel count, and get the rate and channel count on the `AudioData`.

**Pull request merged**